config/default.yml

```diff
@@ -1735,6 +1735,11 @@ Lint/RandOne:
   Enabled: true
   VersionAdded: '0.36'
 
+Lint/ReduceAccumulator:
+  Description: Checks for `reduce` or `inject` blocks that do not update the accumulator each iteration.
+  Enabled: pending
+  VersionAdded: '1.1'
```

Comment:
@bbatsov not sure if this is the correct version now.

Reply:
It is.